docs/app/Examples/modules/Dropdown/index.js

```diff
@@ -43,9 +43,14 @@ const DropdownExamples = () => (
     <ExampleSection title='States'>
       <ComponentExample
         title='Disabled'
-        description='A disabled dropdown menu or item does not allow user interaction'
+        description='A disabled dropdown menu does not allow user interaction'
         examplePath='modules/Dropdown/States/Disabled'
       />
+      <ComponentExample
+        title='Disabled Item'
+        description='A disabled dropdown item does not allow user interaction'
+        examplePath='modules/Dropdown/States/DisabledItem'
+      />
```

Comment:
Let's update the Disabled description to include "menu or item" and drop the title/description on the disabled item example.  This way they show as two examples under one heading:

![image](https://cloud.githubusercontent.com/assets/5067638/18395063/aab83bdc-7671-11e6-9aa4-6c53ba1a9d42.png)